### ISSUE TITLE
[g5k] Support for subnets

### ISSUE THREAD

**Issue by msimonin** [BeyondTheClouds/enoslib#34]
Currently g5k subnets aren't supported meaning :  
- There is no way to reserve one
- There is no way to get the subnet info

At a first sight we could use a special type of g5k network to specify a subnet to reserve : 

```
networks:
  - type: subnet
     slash: 22
     roles: [] # doesn't make really sense here
```

**Comment by msimonin** [comment]
Support for subnets is now merged see #36.
For the record here #39 allows non deploy job to be used.

For you @quentinLeDilavrec, that means that theorically you don't need to use kadeploy anymore for starting VMs on G5K.

**Comment by msimonin** [comment]
closing as #36 has been merged
